Using the version of autoconf in poky results in:

configure.ac:11: error: m4_divert_push: cannot change diversion to `GROW' inside m4_expand

without this patch.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur rpm-5.1.10.orig/configure.ac rpm-5.1.10/configure.ac
--- rpm-5.1.10.orig/configure.ac	2010-08-20 17:12:16.000000000 -0500
+++ rpm-5.1.10/configure.ac	2010-09-03 11:34:44.722097327 -0500
@@ -8,7 +8,7 @@
 
 AC_PREREQ(2.60)
 AC_DEFUN([PACKAGE_BUGREPORT_DEFAULT], [rpm-devel@rpm5.org])
-AC_INIT(rpm, [5.1.10], [PACKAGE_BUGREPORT_DEFAULT])
+AC_INIT(rpm, [5.1.10])
 PACKAGE_TIMESTAMP="2008" dnl # YYYY[-MM[-DD[ HH[:MM]]]]
 AC_MSG_TITLE([RPM Package Manager (RPM)], [$PACKAGE_VERSION])
 
